whois-nrtm4/src/test/java/net/ripe/db/nrtm4/NrtmFileUtilTest.java

```diff
@@ -0,0 +1,70 @@
+package net.ripe.db.nrtm4;
+
+import net.ripe.db.nrtm4.dao.NrtmDocumentType;
+import net.ripe.db.nrtm4.dao.NrtmSource;
+import net.ripe.db.nrtm4.dao.NrtmVersionInfo;
+import net.ripe.db.nrtm4.domain.PublishableSnapshotFile;
+import net.ripe.db.nrtm4.util.NrtmFileUtil;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Set;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.startsWith;
+
+
+public class NrtmFileUtilTest {
+
+    private final NrtmVersionInfo testSnapshotVersion = new NrtmVersionInfo(
+        21L,
+        new NrtmSource("TEST"),
+        22L,
+        "1234567890abcdef",
+        NrtmDocumentType.SNAPSHOT,
+        123123
+    );
+
+    private final NrtmVersionInfo testDeltaVersion = new NrtmVersionInfo(
+        21L,
+        new NrtmSource("TEST"),
+        22L,
+        "1234567890abcdef",
+        NrtmDocumentType.DELTA,
+        123123
+    );
+
+    @Test
+    void snapshot_file_name_looks_legit() {
+        final var file = new PublishableSnapshotFile(testSnapshotVersion);
+        final var name = NrtmFileUtil.newFileName(file);
+        assertThat(name, startsWith("nrtm-snapshot.22."));
+    }
+
+    @Test
+    void session_ids_are_unique() {
```

Comment:
Is this test necessary? it's testing the external `UUID.randomUUID()` method (it may return a duplicate at any time).